cmd/subnetcmd/deploy.go

```diff
@@ -326,17 +316,29 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 
 	// deploy to public network
 	deployer := subnet.NewPublicDeployer(app, useLedger, kc, network)
-	subnetID, blockchainDeployed, blockchainID, tx, err := deployer.Deploy(controlKeys, subnetAuthKeys, threshold, chain, chainGenesis)
+	isFullySigned, subnetID, blockchainID, tx, err := deployer.Deploy(controlKeys, subnetAuthKeys, threshold, chain, chainGenesis)
 	if err != nil {
 		return err
 	}
 
-	if !blockchainDeployed {
-		if err := saveTxToDisk(tx, outputTxPath); err != nil {
+	if !isFullySigned {
+		remainingSubnetAuthKeys, err := txutils.GetRemainingSigners(tx, network, subnetID)
+		if err != nil {
 			return err
 		}
-		remainingSubnetAuthKeys, err := getTxRemainingSigners(tx, network, subnetID)
-		if err != nil {
+		signedCount := len(subnetAuthKeys) - len(remainingSubnetAuthKeys)
```

Comment:
This code looks duplicated between the two methods, should we move it into a shared function?